builds/jenkins-build-pack-website/Jenkinsfile

```diff
@@ -200,7 +200,10 @@ node()  {
 			if(service_config['framework'] == 'angular' || service_config['framework'] == 'react') {
 				events.sendStartedEvent('BUILD', 'build starts', context_map, environment_logical_id)
 				try {
-					buildWebsite(service_config['framework'])
+					def packageJson = readFile("app/package.json")
+					def angularVersionData = new groovy.json.JsonSlurperClassic().parseText(packageJson).dependencies.'@angular/core'
+					def version = angularVersionData.substring(1, angularVersionData.indexOf('.'))
```

Comment:
What happens if the build agent i.e. global angular cli has a different version of angular cli? Isn't that what counts here as we are using `ng build` for the build which uses global angular cli?

Reply:
yes but in order to support the older services with the same build pack, if else condition was introduced for building older and newer angular versions of websites.

Reply:
Now, if the angular cli in the package.json is 5.x, the angular cli in global is 7.x, u will be running with a new version of angular in global along with the --prod flag which will fail. Right?